pages/linux/semanage-port.md

```diff
@@ -16,6 +16,10 @@
 
 `sudo semanage port {{-a|--add}} {{-t|--type}} {{ssh_port_t}} {{-p|--proto}} {{tcp}} {{22000}}`
 
+- Add a user-defined rule that assigns a label to a protocol-port-range pair:
```

Comment:
I suppose the "more technically correct" thing to do here is to use an en dash, but that makes it non-ascii which may be less than ideal.

```suggestion
- Add a user-defined rule that assigns a label to a protocol–port-range pair:
```

Reply:
@gutjuri @spageektti Do we have a policy involving the use of em dash vs en dash vs hyphen?